src/test/java/racing/domain/CarTest.java

```diff
@@ -0,0 +1,44 @@
+package racing.domain;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+class CarTest {
+    @Test
+    @DisplayName("정상적인 자동차 객체 생성")
+    void createCar() {
+        assertDoesNotThrow(() -> new Car("pobi"));
+    }
+
+    @Test
+    @DisplayName("자동차 이름이 5자 초과이면 예외")
+    void createCarWithOverSize() {
+        assertThatThrownBy(() -> new Car("pobipobi"))
+                .isInstanceOf(IllegalArgumentException.class);
+    }
+
+    @Test
+    @DisplayName("자동차 이름이 공백이면 예외")
+    void createCarWithEmpty() {
+        assertThatThrownBy(() -> new Car(""))
+                .isInstanceOf(IllegalArgumentException.class);
+    }
+
+    @ParameterizedTest
+    @ValueSource(ints = {4, 9})
+    @DisplayName("랜덤값이 4이상이면 자동차를 전진합니다")
+    void moveForward(int randomNumber) {
+        Car car = new Car("pobi");
+        car.moveByNumber(randomNumber);
+        String[] split = car.toString().split(" : ");
+        int position = split[1].length();
+        assertEquals(1, position);
```

Comment:
이렇게도 써볼 수 있어요. 수정할 필요는 없습니다!
```suggestion
        assertThat(position).isEqualTo(1);
```
`assertEquals()` 와 `assertThat().isEqualTo()`, assertJ 라이브러리에 대해 한 번 알아보시고 사용해보시는 것도 추천해요!
assertJ 는 특히나 쉽고 나름 유쾌한 공식문서를 가지고 있어서, 처음으로 공식문서를 읽어보기 좋은 입문용 공식문서입니다 :)

Reply:
피케이의 리뷰를 받고 의식 없이 AssertJ와 JUnit을 혼용해서 썼다는 것을 알게 되었어요 😅
해당 코드와 함께 다른 테스트 코드에도 AssertJ를 사용해보았습니다

AssertJ 공식 문서도 꼭 읽어볼게요! 추천해주셔서 감사합니다